asana2sql/fields_test.py

```diff
@@ -20,6 +20,7 @@ def test_assignee_field(self):
 
         ws.add_user.assert_called_once_with(user)
 
+
```

Comment:
Really, this would be the only thing that I'd have a question about (and it's probably too early in asana2sql to be concerned about style guide things), but do we have a rule about extra whitespace lines such as this? (also: this is super trivial, but let's see what comments inside of a diff do with the Asana/GH integration.)